### PR TITLE
Add background shop

### PR DESCRIPTION
**Pull request by kaikecarlos**

### CODE REVIEW

File: src/pages/BackgroundShop.vue [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                    <div class="card-image">
+                        <figure class="image is-4by3">
+                            <img :src="background.fullSizeURL" alt="Image">
+                        </figure>
+                    </div>
+                    <div class="card-content">
+                        <div class="media-content">
+                            <p class="title is-4">{{ background.displayName }}</p>
+                            <p class="subtitle is-6">{{ background.description }}</p>
+                            <p class="subtitle is-5">Price: {{ background.price }}</p>
```
Comment (perronosaurio) [inline src/pages/BackgroundShop.vue]:
```suggestion
                            <p class="subtitle is-5">Price: {{ background.price }} Switchcoins</p>
```

File: src/components/Navbar.vue [hunk 2]
```diff
@@ -59,11 +59,12 @@
 </template>
 
 <script>
+
 export default {
   name: 'navbar',
   data () {
     return {
-      discord: this.$discord.state
+      discord: this.$discord.state,
```
Comment (dsgpaiva) [inline src/components/Navbar.vue:67]: Unnecessary comma (,)

File: src/pages/BackgroundShop.vue [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            <h1 class="title">
+                Background Shop
+            </h1>
+            Buy new backgrounds!
+            </div>
+        </div>
+        </section>
+        <section class="section">
+            <div class="box text-centered" >
+                <h1 class="coins">Your SwitchCoins: 0 </h1>
```
Comment (dsgpaiva) [inline src/pages/BackgroundShop.vue:15]: Switchcoins*

File: src/components/Navbar.vue (hunk 2 above)
Comment (dsgpaiva) [inline src/components/Navbar.vue:67]: Unnecessary comma `,`

**Review by dsgpaiva — DISMISSED** [review]

**Comment by beescuit** [comment]
haHaaaa

**Comment by beescuit** [comment]
are we supposed to use 4-space spacing for HTML related stuff?

**Comment by pedrofracassi** [comment]
I don't know, but check the new website repo. This repo will be archived soon.

**Pull request closed without merging**